TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@ericstj it is not about host servicing. It is NETCore.App version that is the problem. Host has its own versioning scheme. @weshaggard does the above sound reasonable to you? We have master 1.0.1, which causes downgrade of rel to 1.0.0-servicing.